src/SapNwRfc/SapFunctionMetadata.cs

```diff
@@ -121,5 +121,8 @@ private ISapExceptionMetadata GetExceptionByName(string name)
 
             return new SapExceptionMetadata(excDesc);
         }
+
+        // Used by SapServer.InstallGenericServerFunctionHandler
+        internal IntPtr GetFunctionDescHandle() => _functionDescHandle;
```

Comment:
I'd like to keep classes closed. In my book, internal properties like this create unwanted coupling. Would there be an other way to solve what you're trying to do?

Reply:
Hm, not without exposing the `_functionDescHandle` in the public interface or some other kind of internal coupling.